src/legacy/core_plugins/kibana/server/tutorials/register.js

```diff
@@ -81,66 +80,66 @@ import { consulMetricsSpecProvider } from './consul_metrics';
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(couchbaseMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(dropwizardMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(elasticsearchMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(etcdMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(haproxyMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(kafkaMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(kibanaMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(memcachedMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(muninMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(vSphereMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(windowsMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(windowsEventLogsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(golangMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(logstashMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(prometheusMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(zookeeperMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(uptimeMonitorsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(cloudwatchLogsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(awsMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(mssqlMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(natsMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(natsLogsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(zeekLogsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(corednsMetricsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(corednsLogsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(auditbeatSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(iptablesLogsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(ciscoLogsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(envoyproxyLogsSpecProvider);
+  server.newPlatform.setup.plugins.home.tutorials.registerTutorial(couchdbMetricsSpecProvider);
   server.registerTutorial(emsBoundariesSpecProvider);
```

Comment:
So `apm` and `ems` tutorials are the only ones still relying on legacy things and can't currently be migrated? Is this in the scope of the PR or will this be done in a next step when the plugins are ready?

Reply:
That will be done as a follow up PR where we move the tutorials themselves into relevant folders. The idea is that teams/apps that own a product will move their tutorial and registration into their plugin. Tutorials that aren't app specific will remain as Kibana-type tutorials and those I'll move to the `home` plugin.